HHS/Head-Start-TTADP
# TTAHUB-416: Updates for Date Picker Accessibility

## Description of change

Update aria messages on Start and End date selectors to include expected format.

Clean up contrast (default, hover, select) on calendar selector arrow buttons.


## How to test

In the application tab or click on any start or end date entry field. We should now tell the user via screen reader the field and expected format.

Open all calendar selectors in the app the month arrow selector buttons should now have a gray contrast with a hover and selection.


## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-416


## Checklists

### Every PR


~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy


### After merge/deploy
